src/pages/minjeong/Login/loginMj.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import './loginMj.scss';
+import { useNavigate } from 'react-router-dom';
+
+function LoginMj() {
+  const navigate = useNavigate();
+  let [id, setId] = useState('');
+  let [pw, setPw] = useState('');
+  let [isValid, setIsValid] = useState(false);
+
+  function saveUserId(event) {
+    setId(event.target.value);
+    console.log('id :', id);
+  }
+
+  function saveUserPw(event) {
+    setPw(event.target.value);
+    console.log('pw :', pw);
+  }
+
+  const goToMain = () => {
+    navigate('/mainmj');
+  };
+
+  const isActive = id.includes('@') && pw.length >= 5;
+
+  return (
+    <div className="Login">
```

Comment:
```suggestion
    <div className="loginMj">
```
가장 부모태그의 className은 컴포넌트의 이름과 동일하게 하되,
소문자로 시작해야 합니다!